src/beamlime/applications/handlers.py

```diff
@@ -135,7 +136,7 @@ def plot_item(self, name: str, data: sc.DataArray) -> None:
             self.artists[name] = next(iter(plot.artists))
             self.figures[name] = plot
         else:
-            figure.update(data, key=self.artists[name])
```

Comment:
is this old plopp code?

Reply:
@nvaytet?

Reply:
Probably...?

Reply:
Yeah, it was...! I'll update the plopp version in the dependencies : D